apis/r/R/SOMADenseNDArray.R

```diff
@@ -35,7 +35,7 @@ SOMADenseNDArray <- R6::R6Class(
     read_arrow_table = function(
       coords = NULL,
       result_order = "auto",
-      log_level = "warn"
+      log_level = "auto"
```

Comment:
Was this meant to be included in this PR?

Reply:
Yes it was -- I should have commented. It actually overrides the environment variable `SPDLOG_LEVEL`. This has already been fixed for other classes except this one. I thought this was a nice little thing to include in this 'misc. bits' PR.

Sorry for the lack of clarity!